libraries/AP_GPS/AP_GPS_UBLOX.h

```diff
@@ -518,7 +521,7 @@ class AP_GPS_UBLOX : public AP_GPS_Backend
     struct PACKED ubx_mon_ver {
         char swVersion[30];
         char hwVersion[10];
-        char extension[50]; // extensions are not enabled
+        char extension[30*UBLOX_MAX_EXTENSIONS]; // extensions are not enabled
```

Comment:
need to test this on a ublox m6

Reply:
Tested. There are no extensions so it works ok